plugin/eqdetect/source/dextool/plugin/eqdetect/subfolder/codegenerator.d

```diff
@@ -51,4 +51,48 @@ class SnippetFinder{
         temp = temp ~ content[mutation.offset_end .. content.length];
         return temp;
     }
+
+    @trusted static auto generateKlee(string[] params, string source_name, string mutant_name, string function_name){
```

Comment:
After reading the function I do not fully understand its intent.
I think 2-3 lines of comments explaining the intent such as what the code in main intends to do.